### PR TITLE
Update snapshot creation workflow

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

File: scripts/snapshot_creator.py [hunk 1]
```diff
@@ -86,17 +100,26 @@ def parse(stream):
                         help='use develop specs for roots and their immediate '
                              'dependencies')
     parser.add_argument('--name', '-n', required=False,
-                        help='naem the environment something other than the '
+                        help='name the environment something other than the '
```
Comment (psakievich) [inline scripts/snapshot_creator.py:111]: Thanks!

File: scripts/snapshot_creator.py [hunk 2]
```diff
@@ -86,17 +100,26 @@ def parse(stream):
...
                         'date')
-    parser.set_defaults(modules=False, use_develop=False, stop_after='install')
+    parser.add_argument('--use_machine_name', '-mn', action='store_true',
```
Comment (psakievich) [inline scripts/snapshot_creator.py:113]: This is a good fix.

File: configs/base/packages.yaml [hunk 3]
```diff
@@ -1,6 +1,4 @@
 packages:
-  amr-wind:
```
Comment (psakievich) [inline configs/base/packages.yaml:2]: I take it isn't intentional to remove these lines?

**Review by psakievich — COMMENTED** [review]
Just the amr-wind packages thing getting fixed and then I'm fine merging

File: configs/base/packages.yaml (hunk 3 above)
Comment (jrood-nrel) [reply configs/base/packages.yaml:2]: My rebase did some unintended things, but the amrex-hydro PR was merged, so this should be fine.

Comment (psakievich) [reply configs/base/packages.yaml:2]: Okay cool

**Review by psakievich — APPROVED** [review]

**Pull request merged**